### DISCUSSION TITLE
Question: How to format how content is displayed

### DISCUSSION THREAD

**Discussion by jlerbsc** [spring-projects/spring-shell#666] [General]
I would like to display markdown content in the console taking into account the format of the document, for example the titles... Can you tell me if this is possible and possibly give me some implementation tips.

**Comment by jvalkeal** [comment]
There is nothing for things like this. Though it's an interesting idea if you could convert markdown/asciidoc files to plain text having some level of formatting. What comes for the actual formatting it depends on a terminal in use. It's all about ansi escape characters and unicode characters what modern terminal supports.

Sample app has some commands to play with those:

![image](https://user-images.githubusercontent.com/50398/218706562-fd1daeeb-162c-40e3-b142-1b774b1d8735.png)
